### ISSUE TITLE
Sortable components page

### ISSUE THREAD

**Issue by oppenheimer-** [codyhouse/codyframe#3]
Dear codyhose team,
i couldn't find a repo for your website, so i hope you don't mind me writing this idea here.

I'd love to sort the [components](https://codyhouse.co/ds/components) by date to stay up too date when you push new components to the list.

**Comment by sebastiano-guerriero** [comment]
Hi! The components are already sorted by 'newest first'. If you see the 'last updated' date changed, but you don't see a new component in the gallery, it's because we've updated an existing component.

**Comment by oppenheimer-** [comment]
Yeah, ok. This could be a beneficial feature then to sort by last update. To know which components received an update in this view. Especially when you marked something as fave. Just brainstorming here. ;)

**Comment by sebastiano-guerriero** [comment]
absolutely! Makes sense 👍

**Comment by TeaEwan** [comment]
I think it's useful to add a date filter, just the year and month. 

The update date is not shown on the component page. Would it be much less work to add only date filtering?

Example: 

<img width="333" alt="image" src="https://github.com/codyhouse/codyframe/assets/93706123/43d960f9-6f96-4e23-afff-e76c93df2b04">

Thank you.

**Comment by sebastiano-guerriero** [comment]
We'll be adding an 'Updated on...' date on the component page, when the component is updated. One thing we're trying to figure out is if there's a way to show the changes, similarly to checking changes on Github. I believe the main problem we should solve is showing what's changed to users that reported a bug. I'm not sure a 'filter by updated date' helps if we don't show what's changed.

**Comment by TeaEwan** [comment]
Because I usually download components and classify them by year and month, to avoid being confused with the old version. Add 'Updated on...' date on the component page is better. 

I usually use "Beyond Compare" app. If add diff, I don't know if the following items are helpful?

```
https://github.com/kpdecker/jsdiff
https://github.com/AsyncBanana/microdiff
```
